actors/runtime/src/test_utils.rs

```diff
@@ -245,6 +245,35 @@ pub struct ExpectComputeUnsealedSectorCid {
     exit_code: ExitCode,
 }
 
+pub fn expect_abort_contains_message(
+    expect_exit_code: ExitCode,
+    expect_msg: &str,
+    res: Result<RawBytes, ActorError>,
+) {
+    let err = res.expect_err(&format!(
+        "expected abort with exit code {}, but call succeeded",
+        expect_exit_code
+    ));
+    assert_eq!(
+        err.exit_code(),
+        expect_exit_code,
```

Comment:
Nit: I usually think of arguments to assertions being `(expected, actual)`. Is there an entrenched Rust idiom of doing it the other way round?

Reply:
Not that I know of.